### DISCUSSION TITLE
Add dark mode to nicegui.io

### DISCUSSION THREAD

**Discussion by suspiciousRaccoon** [zauberzeug/nicegui#973] [Ideas / Feature Requests]
Unless I've missed it the website doesn't have a button or option to switch between light and dark mode. Adding this would be nice for coding during the night, or vampires who hate light 🧛. 
I'm not sure how dark mode is implemented in NiceGUI but it might conflict with the dark mode demo in the docs.

**Comment by falkoschindler** [comment]
Good point, @suspiciousRaccoon!
In fact, there's already a feature request: https://github.com/zauberzeug/nicegui/discussions/771.
Therefore I'd suggest to continue the discussion over there. 🙂
